app/js/lib/foxbox/foxbox.js

```diff
@@ -500,6 +500,12 @@ export default class Foxbox extends Service {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       .then((services) => {
         return services.map((service) => {
+          // Some service don't have name, but can have product_name instead,
+          // eventually we should get rid of this line and use unified service
+          // label field provided by taxonomy.
+          service.properties.name = service.properties.name ||
+            service.properties['product_name'];
```

Comment:
I thought it forbids underscores in the names, but didn't pay attention - let me double check